frontend/src/console/components/NoticeBoard/NoticeBoard.spec.tsx

```diff
@@ -89,6 +89,9 @@ describe('NoticeBoard tests', () => {
     ]
   ];
   const date = new Date('2020-06-13T15:07:05.158Z');
+  // toLocaleDateString will not behave in the same way on each machine.
+  // this will make it consistant everywhere during testing.
+  date.toLocaleDateString = () => "June 13, 2020"
```

Comment:
@shimulch  can you please address this comment? I am doing a conditional approval on this :)

Reply:
@farhaanbukhsh, sorry, I thought I responded on this one, my bad.

`toLocaleDateString` doesn't behave the same everywhere. As the name suggests, it uses local date formatting settings, and in production that is the actual goal of using it. But when doing snapshot testing, my local computer produces `13 June, 2020` rather than `June 13, 2020`. Which is actually fine but causing the test to fail. Thus I've simply mocked it. Since we are using a static date `2020-06-13T15:07:05.158Z`, I think mocking this method is fine. Now, it will test correctly on any machine.

Reply:
this also happens when you use a different node version. Which node version are you using?

Reply:
@farhaanbukhsh, okay. I am on node `v15.8.0`, which is not LTS I guess.

Reply:
@shimulch can you try using node version 12.20 which is LTS also and probably you would not need to make this change. :) can you please try and see.

Reply:
@farhaanbukhsh Yes, it works fine with the earlier version. Reverted my change.